columnflow/production/cms/seeds.py

```diff
@@ -117,6 +118,17 @@ def deterministic_event_seeds(self: Producer, events: ak.Array, **kwargs) -> ak.
     return events
 
 
+@deterministic_event_seeds.init
+def deterministic_event_seeds_init(self: Producer) -> None:
+    """
+    Producer initialization that adds columns to the set of *used* columns based on the
+    *event_columns*, *object_count_columns*, and *object_columns* lists.
+    """
+    # add used columns
+    for column in self.event_columns + self.object_count_columns + self.object_columns:
+        self.uses.add(Route(column))
```

Comment:
This can be simplified

```suggestion
    self.uses.update(self.event_columns, self.object_count_columns, self.object_columns)
```

Reply:
This misses the `Route()` cast.